README.md

```diff
@@ -69,23 +70,18 @@ end
 
 class ResponseHandler
   def call(response, dataset)
-    Array([JSON.parse(response.body)]).flatten
+    Array([JSON.parse(response.body, symbolize_names: true)]).flatten
```

Comment:
Yeah found out a while back, makes things a lot easier than trying to handle it yourself lol